app/scripts/directives/oauth.js

```diff
@@ -67,11 +66,12 @@ directives.directive('oauth', [
         scope.revokePath    = scope.revokePath    || undefined;
         scope.logoutPath    = scope.logoutPath    || undefined;
         scope.template      = scope.template      || undefined; // was default to 'views/templates/default.html';
-        scope.responseType  = scope.responseType  || 'token';
+        scope.responseType  = scope.responseType  || 'id_token token';
         scope.text          = scope.text          || 'Sign In';
         scope.state         = scope.state         || undefined;
         scope.scope         = scope.scope         || undefined;
         scope.storage       = scope.storage       || 'sessionStorage';
+        scope.nonce         = scope.nonce         || 'k9699'; //TODO make this random 5 digits
```

Comment:
I noticed this issue too in the oauth-ng master. I have some idea to fix (standardize) it. We should have coffee together and share ideas if you also plan to fix : ). Anyway, not a blocker for this PR.